SportsHub/SportsHub.Application/Services/ArticleService.cs

```diff
@@ -43,5 +43,22 @@ public async Task<bool> CreateArticleAsync(CreateArticleDTO adminInput)
 
             return true;
         }
+
+        public async Task<bool> EditArticle(CreateArticleDTO adminInput)
+        {
+            var article = GetByTitleAsync(adminInput.Title).GetAwaiter().GetResult();
+
+            if(article == null)
+            {
+                return false;
+            }
+
+            article.CategoryId = adminInput.CategoryId;
```

Comment:
add this into mapper, please

Reply:
Added auto mapper and check if the CategoryId exists.